lib/src/container/store.rs

```diff
@@ -21,7 +21,7 @@ use flate2::Compression;
 use fn_error_context::context;
 use futures_util::TryFutureExt;
 use oci_spec::image::{
-    self as oci_image, Arch, Descriptor, History, ImageConfiguration, ImageManifest,
+    self as oci_image, Arch, Descriptor, Digest, History, ImageConfiguration, ImageManifest,
```

Comment:
We still use `oci_image::Digest` in a bunch of places, not a big deal but I can do a followup PR after this merges just so we're consistent everywhere.